docs/interfaces/json-api.md

```
@@ -138,17 +137,15 @@ on | bool | On/Off state of the light. You can also use `"t"` instead of `true`
 transition | 0 to 65535 | Duration of the crossfade between different colors/brightness levels. One unit is 100ms, so a value of `4` results in a transition of 400ms. Max transition time is 65 seconds = 65535 milliseconds.
 tt | 0 to 65535 | Similar to transition, but applies to just the current API call. Not included in state response. Max transition time is 65 seconds.
 ps | -1 to 250 | ID of currently set preset. `1~17~` can be used to iterate through presets 1-17, or `4~10r` to select random preset between presets 4 and 10 (inclusive).
-~~pss~~ | ~~0 to 65535~~ | ~~Bitwise indication of preset slots (0 - vacant, 1 - written). Always 0 in 0.11. Not changable.~~ _Removed as of v0.11.1_
-psave | 1 to 250 (16 prior to 0.11) | Save current light config (state) to specified preset slot. Not included in state response.
+psave | 1 to 250 | Save current light config (state) to specified preset slot. Not included in state response.
 sb | bool | Used with `psave`. Save segment bounds (`start` & `stop`).
 ib | bool | Used with `psave`. Save [brightness](#bri).
 sc | bool | Used with `psave`. Save [selected segments](#seg-sel).
 pl | -1 to 250 | ID of currently set playlist. _(read-olny)_
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_only; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)